Collibri/ClientApp/src/components/CreateRoom.js

```diff
@@ -1,64 +1,48 @@
-import React, { useState } from "react";
-import { TextField, Button, Modal, Box, Typography, Divider } from "@mui/material";
+import React, {useRef, useState} from "react";
+import {TextField, Button, Typography, Divider} from "@mui/material";
 import CModal from "./CModal";
 import {createRoom} from "../api/LandingPageApi";
 
-const style = {
-    position: 'absolute',
-    top: '50%',
-    left: '50%',
-    transform: 'translate(-50%, -50%)',
-    width: 500,
-    bgcolor: 'background.paper',
-    border: '2px solid #000',
-    boxShadow: 24,
-    p: 4
-};
-
 export const CreateRoom = ({setRooms}) => {
-    const [name, setName] = useState('');
-    const [open, setOpen] = React.useState(false);
+    const nameFieldRef = useRef(null);
+    const [open, setOpen] = useState(false);
+    const [error, setError] = useState(false);
     const handleOpen = () => setOpen(true);
-    const handleClose = () => setOpen(false);
+    const handleClose = () => {
+        setOpen(false);
+        setError(false);
+    }
+
+    const handleOnChange = () => {
+        if (nameFieldRef.current.value.trim() !== '') {
+            setError(false);
+        } else {
+            setError(true);
+        }
+    }
 
     function handleCreateRoom() {
-        if (name === null || name === '') {
+        if (nameFieldRef.current.value.trim() === null || nameFieldRef.current.value.trim() === '') {
```

Comment:
Man rodos cia reiketu tikrint tik value(be trim), nes jei butu null, pats trim() neveiktu ant null'inio stringo